resources/knative-eventing/charts/knative-eventing/templates/pdb.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: eventing-webhook-pdb
```

Comment:
Let's template the name with chart details and add the default labels.

Reply:
As Discussed, kicking out PDB as we don't know how mutating webhooks behave on upscaling so we don't want to increment replcas of the mutating webhook